vpc-native-beta/inputs.tf

```diff
@@ -120,3 +120,7 @@ variable "confidential_nodes_initial_machine_type" {
   description = "Initial node_pool that is removed should get an n2d machine type even though it will get removed after creation."
   default     = "n2d-standard-2"
 }
+
+variable "resource_labels" {
+  description = "Custom resource labels to add to the cluster configuration."
+}
```

Comment:
It's not clear to me what type of var this should be. My guess is a map of key value pairs, but we should define `type` here as well.